coffeecard/CoffeeCard.WebApi/Models/Token.cs

```diff
@@ -18,5 +19,19 @@ public Token()
         public string TokenHash { get; set; }
 
         [ForeignKey("User_Id")] public virtual User User { get; set; }
+
+        public override bool Equals(object? obj)
```

Comment:
How come object is nullable?

Reply:
Dosen't it need to be in case you are trying to compare to null?

Reply:
To my knowledge, the `if (obj is Token newToken)` is null safe

Reply:
In that case, I can remove it I suppose